### ISSUE TITLE
ReferenceError: username is not defined

### ISSUE THREAD

**Issue by peratik** [wmantly/linux-user#22]
Hi

Code is:

```
let linuxUser = require('linux-sys-user').promise();

linuxUser.getUsers().then(users => {
    users.forEach(user => {
        if (user.uid >= 1000) {
            linuxUser.userExpiration({username: user.username}).catch(console.error)
        }
    });
}).catch(console.error)
```

I get error:
```
ReferenceError: username is not defined
    at Object.exports.userExpiration (/home/saeid/usm/node_modules/linux-sys-user/lib/user.js:305:21)
    at internal/util.js:340:30
    at new Promise (<anonymous>)
    at Object.userExpiration (internal/util.js:339:12)
    at /home/saeid/usm/index.js:6:23
    at Array.forEach (<anonymous>)
    at /home/saeid/usm/index.js:4:11
```

Please advice.

Thank you

**Comment by wmantly** [comment]
What OS/version are you using?

**Comment by peratik** [comment]
WSL2 ubuntu 64bit 20

**Comment by wmantly** [comment]
That sir is a bug. Thank you. I will make the changes and push.

Remove https://github.com/wmantly/linux-user/blob/master/lib/user.js#L305

**Comment by wmantly** [comment]
@peratik this has been fixed with #23 and has been published to NPM, version 1.1.7
